core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -5,17 +5,73 @@
         <span translate="I18N_FOOTER_FOLLOW_US"></span>:
       </h4>
     </div>
-    <a href="https://plus.google.com/109898456505810251700/about" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/gplus.png')]>" alt="Google+">
-    </a>
-    <a href="https://www.youtube.com/channel/UC5c1G7BNDCfv1rczcBp9FPw" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/youtube.png')]>" alt="YouTube">
-    </a>
-    <a href="https://www.facebook.com/oppiaorg" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/fb.png')]>" alt="Facebook">
-    </a>
-    <a href="https://twitter.com/oppiaorg" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/twitter.png')]>" alt="Twitter">
-    </a>
+    
+    <div class="oppia-footer-icons">
+
+        <div class="google-plus-color">
+            <a href="https://plus.google.com/109898456505810251700/about" target="_blank"><i class="fa fa-google-plus"></i>
+            <span class="oppia-icon-accessibility-label">Follow us at Google Plus</span>
```

Comment:
@shaz13 For this and the ones below, simplify the messages in the label. Use the text from the alt tags. For example, for this one, the label should simply be "Google Plus."